deploy/kubernetes/README.md

```diff
@@ -0,0 +1,21 @@
+# PD CSI Driver Deployment in Kubernetes
+
+## Kustomize Structure
+
+The current structure for kustomization is as follows
+
+* `base`: it contains the setup that is common to different driver versions.
+  * `controller_setup`: includes cluster setup and controller yaml files.
+  * `node_setup`:
+    * Linux: includes node yaml file and related setting that is only applicable for Linux.
+    * Windows: includes node yaml file and related setting that is only applicable for Windows.
+* `images`: it has a list of images for different versions.
+  * `stable`: currently only has image list for Linux stable version.
+  * `alpha`: besides image list in stable, it also includes Windows image updates.
```

Comment:
This will become out of date once Windows is out of alpha. Should we make this more generic and say it's images used for alpha features? ditto throughout

Reply:
I think Linux and Windows support should be mentioned for each bullet because it's part of the Kustomization complexity, but I'll update it to mention it's more generally used for features in development.